src/Orleans/Streams/Predicates/ExactMatchStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    internal class ExactMatchStreamNamespacePredicate : IStreamNamespacePredicate
```

Comment:
I think we should also have a RegexStreamNameSpacePredicate 
``` 
        internal class RegexStreamNameSpacePredict
        {
            private readonly Regex regex;

            public RegexStreamNameSpacePredict( Regex regex)
            {
                this.regex = regex;
            }

            bool Match(string streamNameSpace)
            {
                return this.regex.IsMatch(streamNameSpace);
            }
        }
```
This will be a very common IStreamNamespacePredicate which many users will be interested in